lib/tapioca/gem/listeners/mixins.rb

```diff
@@ -62,6 +74,40 @@ def add_mixins(tree, mods, mixin_type)
             end
         end
 
+        sig do
+          params(
+            location: String,
+          ).returns(T::Boolean)
+        end
+        def mixed_in_by_gem?(location)
```

Comment:
The method is `mixed_in_by_gem?` but works with a `location` only, which doesn't seem strictly related to mixins, should we rename it `location_in_gem?`?

Or maybe `mixed_in_by_gem?` should take a constant?